### ISSUE TITLE
Unexpected behavior occurs when using Docker volume mount with itzg/minecraft-server

### ISSUE THREAD

**Issue by eyespore** [Skidamek/AutoModpack#301]
### Bug description

This is an unexpected behavior:
I am experiencing an ERROR when using the `itzg/minecraft-server` image with Docker volume mount, but the issue does not occur when mounting a local directory or when running the server directly with Forge.

Screenshot:
![image](https://github.com/user-attachments/assets/854750f5-0a8c-4c9a-a1e1-b8b776019e0c)

Error log:
```
[18:58:58] [AutoModpackCreation-2/ERROR] [AutoModpack/]: File: /mods/automodpack-mc1.19.2-forge-4.0.0-beta18.jar is not in modpack directory: /data/automodpack/host-modpack/main or current working directory: /data
```

> For the full log please view Steps to reproduce

Interestingly, the error **does not** seem to affect the functionality of the game or mod, and everything appears to run normally. However, I am curious about why this error is occurring with the Docker volume mount and whether it might be related to the Docker setup.

### Steps to reproduce

1.Pulling latest `itzg/minecraft-server` image
```bash
docker pull itzg/minecraft-server:latest
```

2.Pulling `docker-compose-forge.yml` for launching forge server
```bash
mkdir ./mc-forge-1.19.2 && cd ./mc-forge-1.19.2
curl https://raw.githubusercontent.com/itzg/docker-minecraft-server/refs/heads/master/examples/docker-compose-forge.yml > ./compose.yml
```

3.Download `automodpack-mc1.19.2-forge-4.0.0-beta18.jar` into `mods` folder
```bash
wget -P ./mods/ https://github.com/Skidamek/AutoModpack/releases/download/v4.0.0-beta18/automodpack-mc1.19.2-forge-4.0.0-beta18.jar
```

4.Modify environment variable "VERSION" of `docker-compose-forge.yml`, using "1.19.2"
My `compose.yml`:
```yaml
services:
  mc:
    image: itzg/minecraft-server
    ports:
      - "25565:25565"
    tty: true
    stdin_open: true
    environment:
      EULA: "TRUE"
      TYPE: FORGE
      DEBUG: "true"
      UID: 1000
      GID: 1000
      VERSION: "1.19.2"
    volumes:
      - mc_forge:/data
      - ./mods:/mods:ro
volumes:
  mc_forge: {}
```

5.Running docker container with docker-compose-forge.yml
```bash
docker compose up
```

6.Receive error log during launching server
```
[19:26:52] [AutoModpackCreation-2/ERROR] [AutoModpack/]: File: /mods/automodpack-mc1.19.2-forge-4.0.0-beta18.jar is not in modpack directory: /data/automodpack/host-modpack/main or current working directory: /data
```

7.Full log:
https://mclo.gs/rngzXym

### Expected behavior

As stated in the previous description, this ERROR log will not appear when mounting a local directory or running the server directly with Forge. The behavior that I expect is (running with local directory mount):
```yaml
services:
  mc:
    image: itzg/minecraft-server
    ports:
      - "25565:25565"
    tty: true
    stdin_open: true
    environment:
      EULA: "TRUE"
      TYPE: FORGE
      DEBUG: "true"
      UID: 1000
      GID: 1000
      VERSION: "1.19.2"
    volumes:
      - ./data:/data  # mount to local directory
```
The error log mentioned above does not show up:
![image](https://github.com/user-attachments/assets/ef8d7b3d-cce5-436e-b3aa-70bfb3a0632c)

Full log:
https://mclo.gs/vTTFANj

### Minecraft & Mod Loader versions

1.19.2 forge 43.4.0

### Minecraft launcher

PCL2

### Operating system

WSL2-Ubuntu20.04

### AutoModpack version

automodpack-mc1.19.2-forge-4.0.0-beta18.jar

### Other information

I tested with `automodpack-mc1.20.1-forge-4.0.0-beta18.jar` and encountered the same log warning with Docker volume mounts.

I am sure I uploaded the automodpack jarfile into `mods` folder, so I am confused why this error got reported

### Check list

- [X] I have verified that the issue persists in the latest version of the mod.
- [X] I have searched the existing issues and confirmed that this is not a duplicate.

**Comment by Skidamek** [comment]
> Interestingly, the error does not seem to affect the functionality of the game or mod, and everything appears to run normally. 

Hm interesting, what happens when you add more mods to modpack/mods folder? do you also get `"Modpack is empty!"` warn and `"Failed to generate modpack!"` error or just `"File: /mods/automodpack-mc1.19.2-forge-4.0.0-beta18.jar is not in modpack directory: /data/automodpack/host-modpack/main or current working directory: /data"`?

**Comment by eyespore** [comment]
Hi! I uploaded `appleskin-forge-mc1.19-2.4.2.jar` into `./mc-forge-1.19.2/mods` folder, then I started the server with the same steps as above. It seems like the content for `appleskin-forge-mc1.19-2.4.2.jar` is generated successfully,  in the meantime `Modpack is empty!"` and `"Failed to generate modpack!` did not show up

However, the original `*.jar is not in modpack directory, ...` error is still present in this test, and specifies both the `automodpack` itself and the newly added `appleskin`.

![image](https://github.com/user-attachments/assets/80504474-62a6-4afa-b465-efe4d513d9dc)

Full log:
https://mclo.gs/g2etLEL

I joined the server for testing and observed that the `appleskin` is also being distributed to the client correctly
![image](https://github.com/user-attachments/assets/00de83e9-c815-4b10-80a9-214303bc71ed)
